src/config.ts

```diff
@@ -0,0 +1,14 @@
+import { Type } from "@sinclair/typebox";
```

Comment:
We can use [`t` from `elysia`](https://elysiajs.com/concept/schema.html#local-schema) which will provide automatic type inference in the routing. For the sake of consistency, we should probably use it across the entire project unless there is any reason not to due to the the drizzle to typebox converter

Reply:
The schema checking already works and shows up in swagger for both body and response objects. `Type` and `t` are actually the same, from the code: 
```
export { Type as t };
```

My problem with importing from elysia is that it doesn't have all the other utility function, only Type. So in other places we'll still need to use a direct typebox import

Reply:
Okay, sounds like a plan 🙌